helm/external-dns-app/values.yaml

```diff
@@ -288,7 +288,7 @@ global:
 
     # global.image.tag
     # When updating tag make sure to also keep appVersion in Chart.yaml in sync.
-    tag: v0.11.0
+    tag: v0.13.2
```

Comment:
unfortunately upgrading external-dns is blocked for now, see https://github.com/giantswarm/giantswarm/issues/22503#issuecomment-1386818879

it seems there is a fix already implemented, but it hasn't been released yet.

Reply:
thx for pointing this out. In our azure environment we didn't faced this issue.

I will revert this change (and overwrite the image-tag through the `config` repo).